quickstart.py

```diff
@@ -186,7 +224,10 @@ def startServers(config):
     ]
     command.extend(configOptions)
     print "Starting SymServer with command: {}".format(command)
-    runDaemon(command)
+    if foregroundSymServer:
+      subprocess.call(command)
+    else:
+      runDaemon(command)
   return True
```

Comment:
I'm not sure that I totally understand honcho, but it doesn't seem like it addresses my needs. I need to have the option of running in the foreground OR background. Running with Docker requires that the process be in the foreground while the container is to remain open. Running without Docker, you probably do not want Snappy to run in the foreground. To me it looks like honcho does not allow for this flexibility.